go.sum

```diff
@@ -503,6 +503,8 @@ github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712 h1:R8gStypOBmpnHEx1q
 github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712/go.mod h1:PYMCGwN0JHjoqGr3HrZoD+b8Tgx8bKnArhSq8YVzUMc=
 github.com/pingcap/dumpling v0.0.0-20200819021059-26975af91c7b h1:RqxrVPu3e+nvwwwqhHedawNWkI/BQF4MgJRC+gcgnfk=
 github.com/pingcap/dumpling v0.0.0-20200819021059-26975af91c7b/go.mod h1:W1F7jfA4o29S7Mn8Gh9U81wvxdmEl72y5SQOqH0eDDg=
+github.com/pingcap/dumpling v0.0.0-20200821062544-fd4568290bec h1:11EBt45gLbPDwIApjTEdERBPFHKNbkwCZF9LpOOg2zU=
```

Comment:
please run `go mod tidy`, maybe we should add this to `Makefile` as we did for terror check.

Reply:
simply add a command in Makefile when build code. 7a72672